### DISCUSSION TITLE
Example

### DISCUSSION THREAD

**Discussion by Luka803** [MarimerLLC/csla#3745] [General]
Hello!
Can someone can tell where can i find example of parent-child relationship and how to work with it: Adding to the database, retrieving, etc... I am using blazor as an UI.

Thanks in advance!

**Comment by rockfordlhotka** [comment]
The ProjectTracker example shows this on the Project side - not done with the Resources side - but you can see the project edit page and implementation.

**Reply by Luka803** [reply]
Thank you!
